src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2376,6 +2406,7 @@ var projectUniqueNamesForBuildIntToUpdate
                 token);
         }
 
+        // Every action execution comes here right?
```

Comment:
nit: probably not needed unless you expand on this

Reply:
Yeah, I meant to remove it. It was part of my "learning" process about the madness that this 4000 line class is. 

I'll do another to make sure I didn't miss anything. 

If you're curious, the answer to the question is yes :D